kustomize/go.mod

```diff
@@ -22,3 +22,5 @@ exclude (
 replace sigs.k8s.io/kustomize/cmd/config v0.8.1 => ../cmd/config
 
 replace sigs.k8s.io/kustomize/api v0.6.2 => ../api
+
+replace sigs.k8s.io/kustomize/kyaml v0.8.1 => ../kyaml
```

Comment:
The module doesn't `require` kyaml.  Does this line have some impact?

Reply:
Yes. I found that I need to add replace to kustomize go.mod to ask Go to use local kyaml when I built kustomize binary.

Reply:
ok, thanks